loader.qml

```diff
@@ -102,41 +102,50 @@ ApplicationWindow {
                 CheckBox {
                     id: roundCheckBox
 
-                    text: qsTr("Round")
+                    text: "\u25EF"
+                    ToolTip.visible: hovered
+                    ToolTip.delay: 600
+                    ToolTip.text: qsTr("Show the watchface as it would look on a round watch")
                     checked: true
                 }
 
                 CheckBox {
-                    id: halfSize
-
-                    text: qsTr("320x320")
-                }
-
-                CheckBox {
-                    id: twelveHourCheckBox
+                    id: ambientCheckBox
 
-                    text: qsTr("12h time")
+                    text: "\u2b24"
```

Comment:
I wonder if a moon character wouldn’t be clearer that this is ambient mode.
I find the two round characters confusing, it sounds like they should be pretty much about the same thing where one is about the shape and one is about the mode.

Reply:
My original intention was to switch both round symbols from the Ambient and Round check boxes to squares when `Round` is unchecked.
Then imo  full black circle or square actually representing the current setting, would have made clearer that the background gets black in using that check box.
But i see you are even about AmbientMode being about more features then just a black background. Even in this simulation context, we hide and dim certain elements. Have not thought of that and indeed now agree that a moon sort of symbol would much better suggest a Mode change.